.github/workflows/ci.yml

```diff
@@ -13,14 +13,22 @@ jobs:
     name: Test Forge / Unit and Integration Tests
     runs-on: ubuntu-latest
     steps:
+      - name: Set NODE_URL
+        if: ${{ github.event.pull_request.head.repo.full_name == 'foundry-rs/starknet-foundry' }}
+        run: echo "NODE_URL=$(echo ${{ secrets.NODE_URL }}:7070)" >> $GITHUB_ENV
+
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@23bce251a8cd2ffc3c1075eaa2367cf899916d84
       - uses: software-mansion/setup-scarb@v1.3.2
       - uses: software-mansion/setup-universal-sierra-compiler@v1
       - run: cargo test --release --lib -p forge
       - run: cargo test --release --bin snforge
-      - run: cargo test --release integration -p forge
+      - name: Run Forge unit and integration tests
+        env:
+          NODE_URL: ${{ secrets.NODE_URL }}
+        run: |
+          SEPOLIA_RPC_URL="${NODE_URL}:7070/rpc/v0_7" cargo test --release integration -p forge
```

Comment:
nit: it should be possible to set this env variable for the whole workflow at the top of this file https://docs.github.com/en/actions/learn-github-actions/variables#defining-environment-variables-for-a-single-workflow

that way we wouldn't have to set it separately per job